src/deployment-guides/tko-on-vsphere-with-tanzu.md

```diff
@@ -32,6 +32,9 @@ Ensure that your environment has the following general requirements:
 
 The following table provides example entries for the required port groups. Create network entries with the port group name, VLAN ID, and CIDRs that are specific to your environment.
 
+> ✅ If you are following this guide in a lab environment and do not have any
```

Comment:
We are working on an alternate path for this type of content, so let's remove this blurb for now.